patches/0007-LPS-124728-Avoid-breaking-IE11.patch

```diff
@@ -0,0 +1,63 @@
+From 77e1a8c4a354df7c8edcc550bec7517571b0b275 Mon Sep 17 00:00:00 2001
+From: Julien Castelain <julien.castelain@liferay.com>
+Date: Fri, 18 Dec 2020 13:46:17 +0100
+Subject: [PATCH] LPS-124728 Avoid breaking IE11
+
+IE11 Doesn't support URL, URLSearchParams and Object.entries
+In theory though we have a pollyfill for these but it seems that
+they aren't loaded when getUrl is executed.
+---
+ core/ckeditor_base.js | 35 ++++++++++++++++++++---------------
+ 1 file changed, 20 insertions(+), 15 deletions(-)
+
+diff --git a/core/ckeditor_base.js b/core/ckeditor_base.js
+index 6d9fcbc03..0db736ef1 100644
+--- a/core/ckeditor_base.js
++++ b/core/ckeditor_base.js
+@@ -171,26 +171,31 @@ if ( !window.CKEDITOR ) {
+ 			 * @returns {String} The full URL.
+ 			 */
+ 			getUrl: function( resource ) {
+-				var resourceUrl = resource.startsWith(this.basePath)
+-					? new URL(resource)
+-					: new URL(this.basePath + resource);
++				// If this is not a full or absolute path.
++				if ( resource.indexOf( ':/' ) == -1 && resource.indexOf( '/' ) !== 0 )
```

Comment:
Excess whitespace here inside the parens, inconsistent with other parts of the patch (I know this is partly resurrecting old code from prior to a previous patch, but we should still be consistent). Likewise, using `==` and `!==` inconsistently; we should in [general favor the "triple"/"strict" variants](https://github.com/liferay/liferay-frontend-projects/issues/14).

(non-blocking feedback)